requests/utils.py

```diff
@@ -245,6 +245,9 @@ def extract_zipped_paths(path):
     archive, member = os.path.split(path)
     while archive and not os.path.exists(archive):
         archive, prefix = os.path.split(archive)
+        if not prefix:
```

Comment:
This explains why, but the code isn't clear as to what's happening or why this fixes the infinite loop. For example, a far superior comment might be:

```
# If we don't check for an empty prefix after the split (in other words, archive = / before and after the split), we _can_ end up in an infinite loop on a rare corner case affecting a small number of users
```

Reply:
Thanks for the feedback, I've updated the comment.